spec/features/unauthenticated_user_browse_by_category_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'unauthenticated user', type: :feature do
+  it 'can browse items by category' do
+    Category.create(title: "Desserts")
+    visit root_path
+
+    expect(current_path).to eq(root_path)
+    expect(page).to have_content("Desserts")
+  end
+
+  it 'can select a category from the menu' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.